text/0118-search-api.md

```diff
@@ -460,14 +460,182 @@ Search queries using `_geoPoint` returns a `_geoDistance` field containing the d
 - Type: Object
 - Required: False
 
-`_formatted` returns highlighted and cropped attributes specified in `attributesToHighlight` and/or `attributesToCrop` of a search result.
+`_formatted` is an object returned in the search response, only if at least one of the following paramaters has been set in the search query:
+- `attributesToHighlight`
+- `attributesToCrop`
+
+If `attributesToHighlight` and `attributesToCrop` are not set, `_formatted` is not returned.
+
+This `_formatted` object will be present in the each returned document in the `hits` field.
+
+Example:
+
+```json
+{
+    "q": "",
+    "attributesToCrop": ["title"]
+}
+```
+
+```json
+{
+    "hits": [
+        {
+            "id": 2,
+            "title": "Pride and Prejudice",
+            "_formatted": {
+                "id": "2",
+                "title": "Pride and Prejudice"
+            }
+        },
+        {
+            "id": 456,
+            "title": "Le Petit Prince",
+            "_formatted": {
+                "id": "456",
+                "title": "Le Petit Prince",
+            }
+        }
+    ],
+    ...
+}
+```
+
+Which attributes are present in `_formatted`?
+
+The `_formatted` object will contain attributes coming from the original document, depending on the parameters the users set during the search query. Indeed, **the attributes present in `_formatted` is the addition of the attributes present in `attributesToRetrieve`, `attributesToHighlight`, and `attributesToCrop`**.
+
+Kmowing the default value of `attributesToRetrieve` is `["*"]` (so all the attributes present in `displayedAttributes`), if no `attributesToRetrieve` are set in the search query, `_formatted` will return all the `displayedAttributes`.
```

Comment:
But that’s not the case 🤔
If you don’t send any `attributesToRetrieve` then your `_formatted` field should be empty no?
```
{ "q": "hello" }
```
Should not return any `_formatted`.

Reply:
It's a clarification about the `_formatted` behavior when not sending `attributesToRetrieve` but sending `attributesToHighlight` or `attributesToCrop`.

Reply:
Will try to clarify this!

Reply:
I did an update, tell me if it's better